### PR TITLE
inspect-builder throws error with missing remote and local builder

### LINKED ISSUE #386: pack inspect-builder <non-existent-tag> does not error
No error when I try to inspect a non-existant or unreachable builder:
```
± zm |master ✓| → pack inspect-builder cloudfoundry/cnb:0.0.40-bionic
Inspecting builder: cloudfoundry/cnb:0.0.40-bionic

REMOTE:
(not present)

LOCAL:
(not present)

```

### PR DESCRIPTION
**Pull request by ktpv**
Signed-off-by: ktpv <ktpv@users.noreply.github.com>

### CODE REVIEW

File: internal/commands/inspect_builder.go [hunk 1]
```diff
@@ -36,28 +36,33 @@ func InspectBuilder(logger logging.Logger, cfg config.Config, client PackClient)
 				imageName = args[0]
 			}
 
+			remoteOutput, remoteWarnings, remoteErr := inspectBuilderOutput(client, cfg, imageName, false)
+			localOutput, localWarnings, localErr := inspectBuilderOutput(client, cfg, imageName, true)
+
+			if remoteOutput == "(not present)" && localOutput == "(not present)" {
```
Comment (jromero) [inline internal/commands/inspect_builder.go]: I would feel more comfortable with a string comparison if it was a constant used inside the `inspectBuilderOutput` function and in this statement.

Alternatively, an additional return value similar to how maps returns a boolean value (`ok`) if it was found would work here.

No strong opinion either way but something should be changed to not just do arbitrary freeform string comparisons IMO.

File: internal/commands/inspect_builder_test.go [hunk 2]
```diff
@@ -60,14 +60,12 @@ func testInspectBuilderCommand(t *testing.T, when spec.G, it spec.S) {
 
 	when("#Get", func() {
 		when("image cannot be found", func() {
-			it("logs 'Not present'", func() {
+			it("logs 'ERROR Unable'", func() {
```
Comment (jromero) [inline internal/commands/inspect_builder_test.go]: This should read `it("errors when no image is found")`.

Comment (jromero) [reply internal/commands/inspect_builder_test.go]: We also need at least one test (preferably two) where we test when only one of the two is returned and ensure it doesn't error.

Comment (ktpv) [reply internal/commands/inspect_builder_test.go]: Done and done after reorganizing test data.

File: internal/commands/inspect_builder.go (hunk 1 above)
Comment (ktpv) [reply internal/commands/inspect_builder.go]: Did the 2nd option and added additional return value 'present'

**Pull request merged**